var/spack/repos/builtin/packages/py-ipdb/package.py

```diff
@@ -26,8 +31,8 @@ class PyIpdb(PythonPackage):
     # However additional dependencies added below were found via testing.
     depends_on("py-setuptools", type="build")
     # ipdb needs iPython and others available at runtime
-    depends_on("py-ipython@0.10.2:", type=("build", "link"))
-    depends_on("py-traitlets", type=("build", "link"))
-    depends_on("py-six", type=("build", "link"))
-    depends_on("py-pexpect", type=("build", "link"))
-    depends_on("py-prompt-toolkit", type=("build", "link"))
+    depends_on("py-ipython@0.10.2:", type=("build", "link", "run"))
+    depends_on("py-traitlets", type=("build", "link", "run"))
+    depends_on("py-six", type=("build", "link", "run"))
+    depends_on("py-pexpect", type=("build", "link", "run"))
+    depends_on("py-prompt-toolkit", type=("build", "link", "run"))
```

Comment:
None of these are required in the latest release, and they should all be type=build/run, not type=build/link/run. The requirements in the latest release are:
```python
          ':python_version > "3.6" and python_version < "3.11"': ['ipython >= 7.31.1', 'tomli', 'decorator'],
          ':python_version >= "3.11"': ['ipython >= 7.31.1', 'decorator'],               
```
Let me know if you have any questions